pkg/kcl/opt.go

```diff
@@ -18,7 +18,8 @@ type Option struct {
 	Err error
 }
 
-func newOption() *Option {
+// NewOption returns a new Option.
+func NewOption() *Option {
```

Comment:
Why change it to public?

Reply:
I need to use this method in `kpm` to construct an empty `Option`.